packages/iov-bcp-types/src/connection.ts

```diff
@@ -135,6 +135,8 @@ export interface BcpQueryTag {
 
 export interface BcpTxQuery {
   readonly id?: TransactionId;
+  /** transaction from or to this address */
```

Comment:
This is a bit of a question. "any transaction" is quite hard

"send transaction to or from this address" is current implementation of `fromOrToTag`
"atomic swap address with this as sender or recipient" would be other tag
"all transactions signed by this address/pubkey" would be another tag
etc.

I would update the comment to `"send transaction to or from this address"` and maybe modify the name to match

Reply:
Thanks for bringing this up. I missed that since the 3 other blockchain implementations do more or less only support sent transactions. Will adapt. I think the term "address" is not adding value as it just describes the type of the identity, not the relation of the identity to the transaction. I like:

```
readonly sentFromOrTo?: Address;
readonly signedBy?: Address;
```

Reply:
:+1:
That is nice.

Even if `signedBy` is only supported by bns right now, it should actually be doable in most chains, and is a useful thing to query

Reply:
Renamed: https://github.com/iov-one/iov-core/pull/710/commits/dc6ba49cd43dc9ab67e9dea8417fc374e999bea5